src/adaptation/Answer.h

```diff
@@ -31,13 +31,13 @@ class Answer
 
     static Answer Error(bool final); ///< create an akError answer
     static Answer Forward(Http::Message *aMsg); ///< create an akForward answer
-    static Answer Block(const String &aRule); ///< create an akBlock answer
+    static Answer Block(const SBuf &aRule); ///< create an akBlock answer
 
     std::ostream &print(std::ostream &os) const;
 
 public:
     Http::MessagePointer message; ///< HTTP request or response to forward
-    String ruleId; ///< ACL (or similar rule) name that blocked forwarding
+    std::optional<SBuf> ruleId; ///< ACL (or similar rule) name that blocked forwarding
```

Comment:
This field will have a value for Blocked() Answers but not for other Answers, right?

Reply:
Correct.